Buildings/Experimental/DHC/CentralPlants/Cooling/Subsystems/CoolingTowerParellel.mo

```diff
@@ -0,0 +1,296 @@
+    "Fan power"
+    annotation (Dialog(group="Fan"));
+  Medium.ThermodynamicState sta_a=Medium.setState_phX(
+    port_a.p,
+    noEvent(
+      actualStream(
+        port_a.h_outflow)),
+    noEvent(
+      actualStream(
+        port_a.Xi_outflow))) if show_T
+    "Medium properties in port_a";
+  Medium.ThermodynamicState sta_b=Medium.setState_phX(
+    port_b.p,
+    noEvent(
+      actualStream(
+        port_b.h_outflow)),
+    noEvent(
+      actualStream(
+        port_b.Xi_outflow))) if show_T
+    "Medium properties in port_b";
+  Modelica.Fluid.Interfaces.FluidPort_a port_a(
+    redeclare final package Medium=Medium)
+    "Fluid connector a (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{-110,-10},{-90,10}})));
+  Modelica.Fluid.Interfaces.FluidPort_b port_b(
+    redeclare final package Medium=Medium)
+    "Fluid connector b (positive design flow direction is from port_a to port_b)"
+    annotation (Placement(transformation(extent={{90,-10},{110,10}})));
+  Modelica.Blocks.Interfaces.BooleanInput on[num]
+    "On signal for cooling towers"
+    annotation (Placement(transformation(extent={{-140,40},{-100,80}})));
+  Modelica.Blocks.Interfaces.RealInput uFanSpe(
+    final unit="1")
+    "Fan speed control signal"
+    annotation (Placement(transformation(extent={{-140,0},{-100,40}})));
+  Modelica.Blocks.Interfaces.RealInput TWetBul(
+    final unit="K",
+    displayUnit="degC")
+    "Entering air wetbulb temperature"
+    annotation (Placement(transformation(extent={{-140,-80},{-100,-40}})));
+  Modelica.Blocks.Interfaces.RealOutput PFan[num](
+    each final quantity="Power",
+    each final unit="W")
+    "Electric power consumed by fan"
+    annotation (Placement(transformation(extent={{100,50},{120,70}})));
+  Modelica.Blocks.Interfaces.RealOutput TLvg[num](
+    each final unit="K",
+    each displayUnit="degC")
+    "Leaving water temperature"
+    annotation (Placement(transformation(extent={{100,20},{120,40}})));
+  replaceable Buildings.Fluid.HeatExchangers.CoolingTowers.Merkel cooTow[num](
+    each final ratWatAir_nominal=ratWatAir_nominal,
+    each final TAirInWB_nominal=TAirInWB_nominal,
+    each final TWatIn_nominal=TWatIn_nominal,
+    each final TWatOut_nominal=TWatIn_nominal-dT_nominal,
+    each final PFan_nominal=PFan_nominal)
+    constrainedby Buildings.Fluid.HeatExchangers.CoolingTowers.BaseClasses.CoolingTower(
+      redeclare each final package Medium=Medium,
+      each show_T=show_T,
+      each final m_flow_nominal=m_flow_nominal,
+      each final dp_nominal=dp_nominal,
+      each final energyDynamics=energyDynamics)
+    "Cooling tower type"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  Buildings.Fluid.Actuators.Valves.TwoWayEqualPercentage val[num](
+    redeclare each final package Medium=Medium,
+    each final m_flow_nominal=m_flow_nominal,
+    each final dpValve_nominal=dp_nominal)
```

Comment:
`dp_nominal` is applied both at the cooling towers and valves. This is misleading, since the actually pressure drop will be 2x what is expected by a user. 
I'd suggest:
- Create a new `parameter Modelica.SIunits.PressureDifference dpValve_nominal=6000`, and propagate `each final dpValve_nominal=dpValve_nominal` within the `val` block
- Set `each final dpFixed_nominal=dp_nominal` within `val`
- Set `each final dp_nominal=0` within `cooTow`

This will be the most numerically efficient (please let me know if you have any question).